### ISSUE TITLE
UseInAlphabeticalOrder not behaving as expected

### ISSUE THREAD

**Issue by D4rkMindz** [cakephp/cakephp-codesniffer#246]
# Bug Report

**Current behavior:**
There are some style conflicts while using the `cakephp/cakephp-codesniffer` library in combination with e.g. [PHPStorm](https://www.jetbrains.com/phpstorm/). 
PHPStorm formats all imports in alphabetical order as displayed below. (even tested this with excel, its formatted "the same way").

```php
<?php
namespace App\Controller;

// those lines are important
use App\Service\UserService;
use App\Service\BookService;
use App\ServiceHelper\UserService;

class UserController extends AppController 
{
    // some code here ...
}
```
_I do not know how it is with e.g. Eclipse or Atom, didn't test it._

After formatting my code, i usually run a check with `phpcs`. PHPCS now indicates me an error:
The sniff `CakePHP.Formatting.UseInAlphabeticalOrder.UseInAlphabeticalOrder` is triggering an error on the `use` lines.
To "mute" this sniff, i have to format the code as shown below:

```php
<?php
namespace App\Controller;

// those lines are important
use App\ServiceHelper\UserService;
use App\Service\UserService;
use App\Service\BookService;

class UserController extends AppController 
{
    // some code here ...
}
```
**Expected behavior:**
In the above example, PHPCS defines that a `H` comes before `\` (which is correct, but does not correspond to the "natural" sorting of PHPStorm).
It would be expected that the code already correctly formatted by PHPStorm does not throw an error.

```php
<?php
namespace App\Controller;

// those lines are important, it should be like that.
use App\Service\UserService;
use App\Service\BookService;
use App\ServiceHelper\UserService;

class UserController extends AppController 
{
    // some code here ...
}
```

**Where is the bug:**
After a short search I found out in the [`UseInAlphabeticalOrder.php`](https://github.com/cakephp/cakephp-codesniffer/blob/master/CakePHP/Sniffs/Formatting/UseInAlphabeticalOrderSniff.php#L75) Sniff that [`natcasesort()`](https://www.php.net/manual/en/function.natcasesort.php) is used. This sorting leads to the error.

**My proposal:**
I think that the sorting of PHPStorm (which unfortunately cannot be changed) should be accepted and taken as default.
I know that this adjustment is associated with many bugs in existing CakePHP projects, but I think it's not just me. The sorting was already mentioned in 2013 (see #23). However, nothing was ever done.

**Comment by dereuromark** [comment]
Is that in master?
Please see https://github.com/cakephp/cakephp-codesniffer/issues/175 discussion and if this is already fixed in next branch.

**Comment by D4rkMindz** [comment]
@dereuromark That is in the master branch and already fixed in the next branch.
Sorry for bothering
